server/planning/assignments/assignments.py

```diff
@@ -983,10 +983,11 @@ async def update_assignment_on_archive_operation(self, updates, original, operat
                 updated_assignment = self._set_user_for_assignment(
                     assignment, None, assignment_update_data.get("item_user_id")
                 )
-                updated_assignment.get("assigned_to")["desk"] = assignment_update_data.get("item_desk_id")
-                updated_assignment.get("assigned_to")["assignor_user"] = assignment_update_data.get("item_user_id")
 
-                if not assignment_allows_multiple_content_linked(assignment):
+                if assignment_allows_multiple_content_linked(assignment):
```

Comment:
shouldn't this be with `not` so it would only change desk/stage if it doesn't have multiple assignments?